recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,18 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
```

Comment:
Add a docstring to the 'recommend_popular_movies' function to describe what it does and what it returns.